procedures/unit-testing-basics.ipf

```diff
@@ -1238,8 +1245,420 @@ static Function ExecuteHooks(hookType, hooks, juProps, name, procWin, [param])
 	endswitch
 End
 
+/// @brief Background monitor of the Unit Testing Framework
+Function $"UTFBackgroundMonitor"(s)
```

Comment:
@MichaelHuth I'm suprised that this works. I would be in favour of not doing it though.

Reply:
My original idea was to define a StrConstant for the function name as it is in RunTest also checked as the caller function name. However Igor does not take that.